fmn/rules/utils.py

```diff
@@ -79,10 +79,49 @@ def get_packagers_of_package(config, package):
         _cache.configure(**config['fmn.rules.cache'].copy())
 
     key = cache_key_generator(get_packagers_of_package, package)
-    creator = lambda: _get_pkgdb2_packagers_for(config, package)
+    creator = lambda: _get_packagers_for(config, package)
     return _cache.get_or_create(key, creator)
 
 
+def _get_packagers_for(config, package):
+    if config['fmn.rules.utils.use_pagure_for_ownership']:
+        return _get_pagure_packagers_for(config, package)
+    else:
+        return _get_pkgdb2_packagers_for(config, package)
+
+
+def _get_pagure_packagers_for(config, package):
+    log.debug("Requesting pagure packagers of package %r" % package)
+    default = 'https://src.fedoraproject.org/pagure/api'
+    base = config.get('fmn.rules.utils.pagure_api_url', default)
+
+    # XXX. give a default namespace if one is not provided
+    # Later, we need to come back in here and make all of this namespace-aware.
+    if '/' not in package:
+        package = 'rpms/' + package
+
+    url = '{0}/0/{1}'.format(base, package)
+    log.info("hitting url: %r" % url)
```

Comment:
It would be nice if this was a little more explicit so it's clear why the URL is being queried. Something like ``log.info("Querying Pagure at %s for packager information", url)``